templates/activemq.xml.erb

```diff
@@ -128,13 +136,13 @@
           <systemUsage>
             <systemUsage>
                 <memoryUsage>
-                    <memoryUsage percentOfJvmHeap="70" />
+                    <memoryUsage percentOfJvmHeap="75" />
                 </memoryUsage>
                 <storeUsage>
-                    <storeUsage limit="100 gb"/>
+                    <storeUsage limit="150 gb"/>
                 </storeUsage>
                 <tempUsage>
-                    <tempUsage limit="50 gb"/>
+                    <tempUsage limit="25 gb"/>
```

Comment:
This is used for non-persistent messages (hence temp storage rather than regular storage above). EP is not using non-persistent messages so we can reduce this bucket and increase the other.

We have 200gb provisioned on the EC2 instances so these changes move from using up to 150gb (but in reality only 100gb since temp is not used) to 175gb (reality 150gb) which should give us enough disk space for the OS etc since UAT Authoring currently has 187gb free.